### ISSUE TITLE
examples from the validation set

### ISSUE THREAD

**Issue by SorenLi-16** [lukun199/SemanticRL#2]
Dear author, I am very interested in your work! I have finished training the model. Now I want to know how to get the **decoded results from the validation set** as described in IV.D._A Closer Look at the Learned Semantic Representation._ in your paper.
Best wishes!

**Comment by lukun199** [comment]
Hi SongGeGeGe, first thank you for your interest in our work! I have uploaded the inference and evaluation codes to fully reproduce the results in the paper. Please find more details in .`/Evaluation`.

**Comment by SorenLi-16** [comment]
Hi lukun199, I really appreciate your support! Your inspiring work helps me a lot. 

Thanks!
